pkg/sqlcache/informer/listoption_indexer_test.go

```diff
@@ -15,12 +15,16 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/rancher/steve/pkg/sqlcache/db"
 	"github.com/rancher/steve/pkg/sqlcache/partition"
 	"github.com/stretchr/testify/assert"
 	"go.uber.org/mock/gomock"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/sets"
+	//"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
```

Comment:
Do we still need these commented imports?

Reply:
No, sorry for letting that slip in.

Addressed in e966723c